app-crypt/princeprocessor/princeprocessor-0.22-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit edo toolchain-funcs
+
+DESCRIPTION="Standalone password candidate generator using the PRINCE algorithm"
+HOMEPAGE="https://github.com/hashcat/princeprocessor"
+SRC_URI="https://github.com/hashcat/princeprocessor/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}/src"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_compile() {
+	edo $(tc-getCC) -W -Wall -std=c99 ${CFLAGS} ${LDFLAGS} -m64 -DLINUX -o ${PN} pp.c mpz_int128.h
```

Comment:
Isn't ```-m64``` going to be problem on x86?

Reply:
You're right. Sorry, didn't notice that.
I've removed this flag now, since in other ebuilds we usually remove it too.